src/lib.rs

```diff
@@ -18,15 +18,19 @@ use bytemuck::{
   Pod, PodCastError, Zeroable,
 };
 
+#[cfg(target_arch = "bpf")]
+use solana_program::{clock::Clock, sysvar::Sysvar};
+
 solana_program::declare_id!("PythC11111111111111111111111111111111111111");
 
-pub const MAGIC          : u32   = 0xa1b2c3d4;
-pub const VERSION_2      : u32   = 2;
-pub const VERSION        : u32   = VERSION_2;
-pub const MAP_TABLE_SIZE : usize = 640;
-pub const PROD_ACCT_SIZE : usize = 512;
-pub const PROD_HDR_SIZE  : usize = 48;
-pub const PROD_ATTR_SIZE : usize = PROD_ACCT_SIZE - PROD_HDR_SIZE;
+pub const MAGIC            : u32   = 0xa1b2c3d4;
+pub const VERSION_2        : u32   = 2;
+pub const VERSION          : u32   = VERSION_2;
+pub const MAP_TABLE_SIZE   : usize = 640;
+pub const PROD_ACCT_SIZE   : usize = 512;
+pub const PROD_HDR_SIZE    : usize = 48;
+pub const PROD_ATTR_SIZE   : usize = PROD_ACCT_SIZE - PROD_HDR_SIZE;
+pub const MAX_SEND_LATENCY : u64   = 25; 
```

Comment:
@jayantk 
It is the name core program used, what do you think on using MAX_SLOT_DIFFERENCE instead?